workspace/path_type.go

```diff
@@ -28,16 +28,12 @@ func DetectPathType(path string) (PathType, error) {
 		}
 	}
 
-	// If it doesn't exist, then it's an exercise name.
-	// We'll have to walk the workspace to find it.
-	if _, err := os.Stat(path); err != nil {
```

Comment:
`err` is `GetFileAttributesEx a-dir: The system cannot find the file specified.` with a symlinked dir on Windows, and `Lstat` behaves the same with non-existent paths.